server/jetty-app/build.gradle

```diff
@@ -10,6 +10,7 @@ configurations {
 dependencies {
     implementation project(':server-jetty')
 
+    runtimeOnly project(':extensions-performance')
```

Comment:
Given that server already has a dependency on this, we don't need to specify this dependency here.